src/main/java/org/reaktivity/nukleus/tls/internal/stream/TlsServerFactory.java

```diff
@@ -412,66 +404,71 @@ private int decodeClientHello(
         int progress,
```

Comment:
We need to rename the decode. How about `decodeInitialHandshake`

Reply:
I was expecting that `decodeClientHello` could be removed and that the `tlsEngine` would be created in reaction to receiving `BEGIN` frame instead.

Reply:
Currently decodeClientHello handles only `server.tlsEngine.beginHandshake();` I thought we need to on the first data frame which contains ClientHello no? Can we initiate beginHandshake right away?

Reply:
Yes, we should call `tlsEngine.beginHandshake()` in `onNetworkBegin`, so `decodeClientHello` can be removed.